src/FlatButton/FlatButton.js

```diff
@@ -258,6 +263,7 @@ class FlatButton extends Component {
     return (
       <EnhancedButton
         {...other}
+        className={className}
```

Comment:
Actually, I'm wondering, do we need to explicit the className property here? If you don't destructure it, it will be inside the `other` object. That would reduce the number of ligne of code.